docker/dbeaver/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM dbeaver/cloudbeaver:23.0.2
```

Comment:
I put everything into a docker folder. Shall we move all docker related stuff here (nginx setup) or should this be moved under config?

Reply:
On the long run, I would say we should use the same approach for all containers.

Separate Dockerfiles are probably easier to understand than mounting configs from the cpmonitor container into other containers. However, building our own images for nginx would mean we have to find another way of configuring nginx differently for different environments than what we do now (mount different config folders).

For now, I'd say let's move the cpmonitor Dockerfile into /docker/cpmonitor/, adjust README and ci.yml, and think about the nginx-setup  later.

Reply:
As long as you run docker build in the repo root and select the file via -f, it's fine: https://stackoverflow.com/a/34300129